### DISCUSSION TITLE
Open link in a new windows

### DISCUSSION THREAD

**Discussion by moblescanbarato** [PrestaShop/PrestaShop#31862] [Ideas]
### The problem

When we show the links to our social networks and the customer clicks, the link opens the social network in the same window, with which the customer leaves the store, something that we do not want to happen

### The solution

I think it's very easy, just add the expression in the link code: target="_blank"

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by FabienPapet** [comment]
Hey @moblescanbarato thank you for the proposition you made. As you said, this is quite simple, maybe you can make a pr for this ?

**Reply by moblescanbarato** [reply]
Hi Fabien, I'm not a programmer, but I don't think it's necessary to make a program for that, just modify the code within the Social Networks module, just one line, rather, add an expression to the link line: "target ="_blank"

**Reply by FabienPapet** [reply]
Sound likes you already know what to do, so you can do it and participate to the open source project :)
